modules/claims_api/spec/requests/v1/forms/526_spec.rb

```diff
@@ -9,14 +9,15 @@
       'X-VA-First-Name': 'WESLEY',
       'X-VA-Last-Name': 'FORD',
       'X-Consumer-Username': 'TestConsumer',
-      'X-VA-Birth-Date': '1986-05-06T00:00:00+00:00',
+      'X-VA-Birth-Date': '1956-05-06T00:00:00+00:00',
```

Comment:
Why are we changing the birthday both here and in the test?

Reply:
@tycol7 We will need to change either the test or the [form_526_json_api](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/requests/v1/forms/526_spec.rb#L36)

Reply:
Got it - makes sense!